README.md

```diff
@@ -0,0 +1,218 @@
+      }
+    }
+    ```
+
+3.  Add a file named __.config__ in the project's root directory, with
+    suitable `repo-owner` and `repo-name` values. For example:
+
+    ```text
+    repo-owner = sanctuary-js
+    repo-name = sanctuary-type-classes
+    ```
+
+4.  Add a file named __.eslintrc.json__ in the project's root directory,
+    with the following JSON configuration:
+
+    ```json
+    {
+      "root": true,
+      "extends": ["./node_modules/sanctuary-style/eslint-es3.json"]
+    }
+    ```
+
+    Change `es3` to `es6` if desired.
+
+## Usage
+
+### `npm test`
+
+Runs [`npm run lint`](#npm-run-lint), then [Mocha][] and [Istanbul][], and
+finally [`npm run doctest`](#npm-run-doctest).
+
+### `npm run lint`
+
+Runs the following linters:
+
+  - [`check-required-files`][]
+  - [`eslint`][]
+  - [`lint-package-json`][]
+  - [`lint-bower-json`][]
+  - [`lint-readme`][]
+  - [`lint-commit-messages`][]
+
+### `npm run doctest`
+
+Runs [`doctest`][] with appropriate arguments.
+
+### `npm run release`
+
+Runs [`xyz`][] with appropriate arguments.
+
+## Configuration
+
+There are two layers of configuration available: variables and custom scripts.
+
+### Variables
+
+Certain variables may be specified in a file named __.config__ in the project's
+root directory. Each line in the file should be a `${name} = ${value}` pair.
+For example:
+
+    repo-owner = sanctuary-js
+    repo-name = sanctuary-type-classes
+
+Many variables have default values and are therefore optional.
+
+| Variable name                 | Default value | Description                                                                   |
+| ----------------------------- | ------------- | ----------------------------------------------------------------------------- |
+| `repo-owner`                  |               | The name of the GitHub user who owns the repository.                          |
+| `repo-name`                   |               | The name of the GitHub repository.                                            |
+| `default-branch`              | `master`      | The name of the repository's default branch.                                  |
+| `min-branch-coverage`         | `100`         | The minimum acceptable branch coverage (as a percentage).                     |
+| `source-files`                | `index.js`    | Space-separated list of filenames. Globbing is supported.                     |
+| `heading-level`               | `4`           | The `<h[1-6]>` level of headings transcribed from `heading-prefix` comments.  |
+| `heading-prefix`              | `#`           | The character which follows `//` to signify a heading to transcribe.          |
+| `comment-prefix`              | `.`           | The character which follows `//` to signify documentation to transcribe.      |
+| `version-tag-prefix`          | `v`           | The prefix of annotated version tags (use _null_ for `X.Y.Z` format).         |
```

Comment:
It's not immediately clear to me what is meant by use _null_. Do I set the value to literal `null`? Or do omit the value part of my expression ?

Reply:
I agree that it's unclear.

I consulted the [Bash Reference Manual][1] many times in the days prior to writing this. It uses _null_, _null string_, and _null value_ to refer to the empty string (in JavaScript parlance).

I'm attempting to convey that a user should add

    version-tag-prefix =

to her __.config__ if she wants to use `X.Y.Z` format rather than `vX.Y.Z` format. The problem with referring to the empty string is it suggests she should add

    version-tag-prefix = ''

instead, which is incorrect.

Can you think of a better explanation?


[1]: https://www.gnu.org/software/bash/manual/bashref.html

Reply:
I went with this:

> The prefix of annotated version tags (use `version-tag-prefix =` for `` ``).

:zap: